ext/standard/tests/mail/mailhog_utils.inc

```diff
@@ -0,0 +1,66 @@
+<?php
+
+function searchEmailByToAddress (string $to)
+{
+    // Search the mail by $to
+    $c = curl_init();
+    curl_setopt($c, CURLOPT_URL, 'http://localhost:8025/api/v2/search?kind=to&query='.$to);
+    curl_setopt($c, CURLOPT_RETURNTRANSFER, true);
+    $res = curl_exec($c);
+    curl_close($c);
```

Comment:
why don't you just use following?
```
$res = file_get_contents("http://localhost:8025/api/v2/search?kind=to&query=$to");
```
that would remove unnecessary dependency on curl...

Reply:
I completely forgot about it...